features/format.feature

```diff
@@ -554,3 +554,13 @@ Feature: Custom formats
         | basic_encrypted |
         | basic_folder    |
         | basic_dayone    |
+
+    Scenario: Export date counts
```

Comment:
This test should run for all journal types. See the test just above this one in the same file for an example.

Reply:
👍

Reply:
Done!